src/libraries/System.Net.Sockets/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -530,6 +530,12 @@ public void SendPacketsElement_FileStreamMultiPart_Success()
         [Fact]
         public void SendPacketsElement_FileStreamLargeOffset_Throws()
         {
+            if (PlatformDetection.IsWindows10Version22000OrGreater)
+            {
+                // [ActiveIssue("https://github.com/dotnet/runtime/issues/58898")]
```

Comment:
Isn't it better to have the [ActiveIssue] attribute and add the platform detection as a [ConditionalFact] attribute?

Reply:
Addressed in d26494f29ab95c247cb6e7798742190ffa7f2366. `SkipTestException` trick somehow became a repo-wide practice for skipping on Windows 11, also not big fan of it.

Reply:
At least for the SslStream I see this is temporary. For me this seems to have less overhead comparing to build new conditional variables. The other part is that we would see it in database and it is visible during test run e.g. serves as reminder as well.

Reply:
> The other part is that we would see it in database and it is visible during test run e.g. serves as reminder as well.

This makes sense, though I prefer to merge the PR as-is for now. We need to deal with #58898 with high priority anyways.